reactioncommerce/reaction
# Remove hardcoded billing and shipping objects

Resolves #2770 

This PR aims to remove hard coded first billing and shipping objects from most code around orders.

### How to test

- Open browser console.
- Go through order creating/completing/processing-as-admin process.
- Make sure everything works as normal, and there are no errors in the browser console and/or server.
